CompanyEmployees.Core.Domain/Entities/Employee.cs

```diff
@@ -20,7 +20,10 @@ public class Employee
         public string? Position { get; set; }
 
         [ForeignKey(nameof(Company))]
```

Comment:
Avoid using public fields; they violate encapsulation and can lead to inconsistent state. This field is also redundant with the `Salary` property and should be removed before merging.

```suggestion
// Removed: public string salary;
```